neqo-transport/src/connection/tests/mod.rs

```diff
@@ -77,6 +77,22 @@ pub fn default_server() -> Connection {
     c
 }
 
+pub fn disallow_zerortt_server() -> Connection {
```

Comment:
One-off things like this don't need functions to be added to the base fixture.  Just include this stuff inline.